### DISCUSSION TITLE
Option to ignore soft line breaks

### DISCUSSION THREAD

**Discussion by gw31415** [mdx-js/mdx#2213] [Ideas] [answered]
Hello, I am hopeful about this project.

When writing markdown in some East-Asian languages, it is more convenient to not consider soft line breaks as spaces.

Example is below:

```md
# テスト

あのイーハトーヴォのすきとおった風、夏でも底に冷たさをもつ青いそら、
うつくしい森で飾られたモリーオ市、郊外のぎらぎらひかる草の波。またそ
のなかでいっしょになったたくさんのひとたち、ファゼーロとロザーロ、
```
<img width="686" alt="image" src="https://user-images.githubusercontent.com/24710985/210822655-49110969-aa00-4942-9cb5-3c7bf795ecb6.png">

[Pandoc](https://pandoc.org) has an option called [`east_asian_line_breaks`](https://pandoc.org/MANUAL.html#extension-east_asian_line_breaks) to achieve this. If enabled, soft line breaks are ignored when a character with a character width of 2 is entered, resulting in output with no extra spaces.

I would appreciate it if you could provide a way to achieve this with this MDX library.

**Comment by wooorm** [comment, marked as answer]
You can create a plugin for this, see: https://mdxjs.com/docs/extending-mdx/#creating-plugins. A remark (or rehype) plugin for this might already exist.

**Reply by gw31415** [reply]
I had missed the method of plugins. It looks like it will work. Thanks!
